src/item.cpp

```diff
@@ -4600,7 +4600,7 @@ int item::lift_strength() const
 
 int item::attack_time() const
 {
-    int ret = 65 + volume() / 62.5_ml + weight() / 60_gram;
+    int ret = 65 + ( volume() / 62.5_ml + weight() / 60_gram ) / count();
```

Comment:
Could 'count' ever be zero?

Reply:
If you see the definition of count() you'd see it's not possible.